janusgraph-berkeleyje/src/test/resources/log4j.properties

```diff
@@ -9,7 +9,8 @@ log4j.appender.A2.Threshold=ALL
 log4j.appender.A2.layout=org.apache.log4j.PatternLayout
 log4j.appender.A2.layout.ConversionPattern=%-4r [%t] %-5p %c %x - %m%n
 
-log4j.rootLogger=WARN, A1, A2
+#log4j.rootLogger=INFO, A1, A2
```

Comment:
@mbrukman For Travis I minimized logging but I also wanted to make it as easy as possible for developers to turn logging back on. The A1/A2 threshold updates here allow logging to be turned on with one line. In particular one could swap the comment between the two rootLogger lines in any module to turn up logging. If possible I'd like the keep the A1/A2 threshold updates but I can go either way on the commented out line (I actually had it out and then added it back right before submitting the PR). Take it out?